### ISSUE TITLE
Update "dialogs" to snackbars and make them less intrusive

### ISSUE THREAD

**Issue by nazrinharris** [nazrinharris/pecunia#112]
Kinda want it to be like the Arc browser. Sliding from the right, on the top left. Ima nail the design in Figma first though.

**Comment by nazrinharris** [comment]
Been looking through a couple snackbar/dialog/toast packages. Tried `bot_toast` but I found the docs to be difficult to work with. Same issue with the package I'm using right now, `flutter_easy_dialogs`. Tried `fluid_dialog`, which seems to work pretty good for dialogs. But I need toasts. Considered `ndialog` but didn't have the best experience when using it with `farmhub` so I maybe not.

Best candidate is `fl_toast`, but it may be too simple? There's technically a sequential toast but that seems to require me to know the amount of toasts that should exist. Not viable. It doesn't seem to have a way to dismiss it. And won't be dismissed when navigating somewhere else. So it's okay, but its a bit too simple. 

So I'd need something similar to `fl_toast`, but more feature-rich.
